src/routes/pay/likecoin.ts

```diff
@@ -286,6 +290,7 @@ likecoinRouter.post('/', async (req, res, next) => {
         displayName: author.displayName,
         userName: author.userName,
       },
+      replyToDonator: draft.replyToDonator,
```

Comment:
This should be a boolean if [email](https://github.com/thematters/matters-email/pull/171/files#diff-05e59776f6f0ad1fb0e23a7340172f31a91a226c41ee17e66d9a52a3eb34611cR113) doesn't use the content of `replyToDonator`, otherwise, please confirm if empty string `''` is work as expected.

Reply:
fixed